### ISSUE TITLE
MFD Pop-Out Window not in "NG" Style

### ISSUE THREAD

**Issue by bluemax24** [SimSolutions1/Issue-Tracker#38]
**Platform**
X-plane 11, Windows 10, DA 40NG, v1.2.0

**Description**
Whereas the cockpit view is displaying the MFD in "NG"-style, that representation is reverted back to "classical" appearance once the MFD is "popped-out" (e.g. manifold pressure i.s.o. load percentage, etc.)

**Comment by bluemax24** [comment]
I clicked on the "MFD" in teh x-Plane main window to create the "pop-out".
I just updated from v1.04 to v1.2.0 this afternoon and was apparently left with a "left-over" pop-out from v1.0.4 (window-title "X-System").

When I tried to reproduce /document the "pop-out" flow I found that it is correcly working in v1.2.0 (window title "G1000 MFD").
I feel the ticket can be closed (I should have tried before creating it, sorry!)
